cengage/react-magma
# fix(Button): Remove animation on `Button` presses.

Issue: #1295

## What I did

Remove animation on `Button` presses.

## Screenshots:


## Checklist 

## How to test

Open docs/storybook -> Open `Button` component -> Click on button -> Check styles and animations.
